File: min_max_sum.py
#!/usr/bin/python3
import sys
from typing import List


def miniMaxSum(arr: List[int]) -> str:
    total = 0
    min_value = sys.maxsize
    max_value = 0
    for number in arr:
        max_value = max(number, max_value)
        min_value = min(number, min_value)
        total += number
    return f"{total-max_value} {total-min_value}"


print(miniMaxSum([1, 2, 3, 4]))

File: min_max_sum.go
package main

import (
	"fmt"
	"math"
)

func miniMaxSum(arr []int) string {
	min := math.MaxInt32
	max := 0
	total := 0
	for _, number := range arr {
		min = int(math.Min(float64(min), float64(number)))
		max = int(math.Max(float64(max), float64(number)))
		total += number
	}
	return fmt.Sprintf("%d %d", total-max, total-min)
}

func main() {
	s := miniMaxSum([]int{1, 2, 3, 4})
	fmt.Println(s)
}
